resources/privacy-and-gdpr.md

```diff
@@ -6,6 +6,251 @@ Before implementing these controls, read the [Adobe Experience Platform Privacy
+#### Java
+
+**Syntax**
+
+```java
+public static void update(final Map<String, Object> consents);
+```
+
+**Example**
+
+```java
+// example 1, updating users collect consent to 'yes'
+final Map<String, Object> collectConsents = new HashMap<>();
+collectConsents.put("collect", new HashMap<String, String>() {
+    {
+        put("val", "y");
+    }
+});
+
+final Map<String, Object> consents = new HashMap<>();
+consents.put("consents", collectConsents);
+
+Consent.update(consents);
+
+// example 2, updating users collect consent to 'no'
+final Map<String, Object> collectConsents = new HashMap<>();
+collectConsents.put("collect", new HashMap<String, String>() {
+    {
+        put("val", "n");
+    }
+});
+
+final Map<String, Object> consents = new HashMap<>();
+consents.put("consents", collectConsents);
+
+Consent.update(consents);
+```
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+#### Swift
+
+**Syntax**
+
+```swift
+static func update(with consents: [String: Any])
+```
+
+**Example**
+
+```swift
+// example 1, updating users collect consent to 'yes'
+let collectConsent = ["collect": ["val": "y"]]
+let currentConsents = ["consents": collectConsent]
+Consent.update(with: currentConsents)
+
+// example 2, updating users collect consent to 'no'
+let collectConsent = ["collect": ["val": "n"]]
+let currentConsents = ["consents": collectConsent]
+Consent.update(with: currentConsents)
+```
+
+#### Objective-C
+
+**Syntax**
+
+```swift
+static func update(with consents: [String: Any])
+```
+
+**Example**
+
+```text
+// example 1, updating users collect consent to 'yes'
+NSDictionary *collectConsent = @{ @"collect": @{@"val": @"y"};
+[AEPMobileEdgeConsent updateWithConsents:@{@"consents": collectConsent}];
+
+// example 2, updating users collect consent to 'no'
+NSDictionary *collectConsent = @{ @"collect": @{@"val": @"n"};
+[AEPMobileEdgeConsent updateWithConsents:@{@"consents": collectConsent}];
+```
+{% endtab %}
+{% endtabs %}
+
+### getIdentities
+
+When using the Edge Network extensions, use the [Identity.getIdentities](../foundation-extensions/identity-for-edge-network/api-reference.md#getidentities) API to retrieve all the identifier data stored locally by the SDK and send this data to your servers.
+
+## Configuration keys
+
+To update the SDK configuration, programmatically, use the following information to change your default consent values. For more information, see [Configuration API reference](../foundation-extensions/mobile-core/configuration/configuration-api-reference.md).
+
+| Key               | Description                                                  |
+| :---------------- | :----------------------------------------------------------- |
+| `consent.default` | Defines the set of default consent preferences for the SDK in XDM format. For more details, see [Privacy/Personalization/Marketing Preferences \(Consents\) Schema](https://github.com/adobe/xdm/blob/master/docs/reference/mixins/profile/profile-consents.schema.md). |
```

Comment:
Same comment here - we don't need to keep the \\ around the parentheses, so I'd just go with "For more details, see [Privacy/Personalization/Marketing Preferences (Consents) Schema]."